.github/workflows/wheels.yml

```diff
@@ -0,0 +1,152 @@
+name: Build and test wheels
+
+on:
+  push:
+    branches:
+      - master
+      - test
+  pull_request:
+  create:
+
+jobs:
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+        wordsize: [32, 64]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install deps
+        env:
+          PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
+        shell: bash
+        run: |
+          set -ex
+          ${PYTHON} -m pip install --upgrade pip
+          ${PYTHON} -m pip install setuptools wheel
+          ${PYTHON} -m pip install -r python/requirements/CI/wheel-build/requirements.txt
+      - name: Build C extension
+        env:
+          PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
+        shell: bash
+        run: |
+          set -ex
+          # Fix symlinks on windows
+          git config core.symlinks true
+          git reset --hard
+          cd python
+          ${PYTHON} -VV
+          ${PYTHON} setup.py build_ext --inplace
+          ${PYTHON} -c "import kastore"
+      - name: Build Wheel
+        env:
+          PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
+        shell: bash
+        run: |
+          set -ex
+          cd python
+          ${PYTHON} setup.py bdist_wheel
+      - name: Install wheel and run tests
+        env:
+          PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
+        shell: bash
+        run: |
+          set -ex
+          ${PYTHON} -m pip install -v kastore --only-binary :all: -f python/dist/ --no-index
+          rm -rf python/kastore python/*.so
+          ${PYTHON} -m nose -v python
+      - name: Upload Wheels
+        uses: actions/upload-artifact@v1
+        with:
+          name: win-wheel-${{ matrix.python }}-${{ matrix.wordsize }}
+          path: python/dist
+
+  OSX:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install deps
+        run: |
+          pip install --upgrade pip
+          pip install setuptools wheel
+          pip install -r python/requirements/CI/wheel-build/requirements.txt
+      - name: Build C extension
+        run: |
+          cd python
+          python -VV
+          python setup.py build_ext --inplace
+          python -c "import kastore"
```

Comment:
Ditto here

Reply:
Fixed in https://github.com/tskit-dev/kastore/pull/118/commits/5a26733548f9b85c0ad7c731efdf72aacc57a92c